packs/pagerduty/actions/resolve_incident.yaml

```diff
@@ -0,0 +1,10 @@
+name: resolve_incident
+runner_type: run-python
+description: Resolve an incident whose key is provided
+enabled: true
+entry_point: resolve_incident.py
+parameters:
+  event_keys:
+    type: array
```

Comment:
Does this work as an array? If I supply a comma separated list, I think I'd run into a schema problem, no?